wagtail_footnotes/apps.py

```diff
@@ -0,0 +1,7 @@
+from django.apps import AppConfig
+
+
+class WagtailFootnotesAppConfig(AppConfig):
+    label = "wagtail_footnotes"
+    name = "wagtail_footnotes"
+    verbose_name = "Wagtail Footnotes"
```

Comment:
Do we still need a change here? ref: https://github.com/torchbox/wagtail-footnotes/pull/38

Reply:
let's do it. Merging #38 mean this PR will have fewer changes

Reply:
Merged